alfresco-transform-tika/alfresco-transform-tika-boot/src/test/java/org/alfresco/transformer/TikaTransformationIT.java

```diff
@@ -72,7 +72,15 @@ public TikaTransformationIT(final Triple<String, String, String> entry)
     {
         sourceFile = entry.getLeft();
         targetExtension = entry.getMiddle();
-        targetMimetype = extensionMimetype.get(entry.getMiddle());
+        //Single test to cover csv-->pdf
```

Comment:
Comment seems to indicate wrong direction ? Test code is pdf->csv ?

Why do we need this special case ?

Reply:
@KikoUK knows more about this

Reply:
Yes, the comment should be pdf-->csv. This case is the cover one the steps in this pipeline -- [libreofficeToPdfBoxViaPdf ](https://issues.alfresco.com/jira/browse/ATS-762?focusedCommentId=789781&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-789781)

Reply:
OK, I updated the comment